### DISCUSSION TITLE
账号nsfw封禁问题

### DISCUSSION THREAD

**Discussion by necroman1** [BurnHair/docs#160] [技术支持]
使用沉浸式翻译功能观看youtube视频被封，是因为视频内容涉及到了nsfw话题？感觉判定有点严格了，请问是否可以解封。
<img width="258" alt="Snipaste_2024-07-14_16-52-57" src="https://github.com/user-attachments/assets/ec0dafb4-ab59-4037-8295-451165532ca8">
<img width="857" alt="Snipaste_2024-07-14_16-54-48" src="https://github.com/user-attachments/assets/ab203948-40d9-4a1c-a6cd-c0afaf6496a5">

**Comment by BennyThink** [comment]
#NSFW 8384，分组vip可能在 NSFW，内容：Translate the following source text to Simplified Chinese Language, Output translation directly without any additional text.
Source Text: 越南最美人妻帶我進她豪宅….原來這樣可少奮鬥30年 | the PERKS of marrying a Vietnamese woman!!
Translated Text:


已经解封但是请注意

**Reply by kityousi** [reply]
所以你把之前内容留存审查的公告删除了，实质上暗地继续进行吗？被站长人品吓晕了，后会无期！
